src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        int figureNumber = 11;
+
+        int figuresNumberInSetOne = 0;
+
+        if (figureNumber % 2 == 0) {
+            figuresNumberInSetOne = figureNumber / 2;
+        } else {
+            figuresNumberInSetOne = figureNumber / 2 + 1;
+        }
+
+        Figure [] result = new Figure[figureNumber];

```

Comment:
Don't use verbs for class/interface names. The name `FigureDraw` suggests a verb, indicating an action, which is not appropriate for a class name. Consider renaming it to something that represents the entity or concept, such as `DrawableFigure` if it represents figures that can be drawn.